Plugins/DotNET/DotNETExports.cpp

```diff
@@ -329,6 +347,24 @@ static const char* StackPopString()
     return strdup(String::ToUTF8(value.CStr()).c_str());
 }
 
+static const char* StackPopRawString()
+{
+    auto vm = Globals::VirtualMachine();
+    ASSERT(vm->m_nRecursionLevel >= 0);
+
+    CExoString value;
+    if (!vm->StackPopString(&value))
+    {
+      LOG_WARNING("Failed to pop string - recursion level %i.", vm->m_nRecursionLevel);
+      return nullptr;
+    }
+
+    static std::string retVal = value.CStr();
```

Comment:
By the way, if the answer to the other question is "strdup() leaks memory, need to use static retVal", then just make the `CExoString` the static variable and avoid this extra copy here.